Geometry/HcalCommonData/src/HcalParametersFromDD.cc

```diff
@@ -160,7 +163,6 @@ bool HcalParametersFromDD::build(const DDCompactView* cpv, HcalParameters& php)
     std::cout << " [" << ++i << "] = " << (*it) / CLHEP::deg;
```

Comment:
@bsunanda , why we are using std::cout even in the debug mode?

Reply:
I have changed a few. The rest will be taken care of when I complete the dd4hep part.